tstyche.config.json

```
@@ -0,0 +1,3 @@
+{
+  "testFileMatch": ["**/types.spec.ts"]
```

Just a tiny change, but it would improve TSTyche performance.

```suggestion
  "testFileMatch": ["test/**/types.spec.ts"]
```